Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/pistols.yml

```diff
@@ -0,0 +1,51 @@
+- type: entity
+  name: Valentine
+  parent: BaseWeaponPistol
+  id: WeaponPistolValentine
+  description: An Old and bloodstain this Pistol is feature with a wooden pistol grip that has S.T.A.R on it
```

Comment:
Just because it is old doesn't mean it still isn't mass produced,  IRL the Mosin is still in the active military still made and produced and still sold.  The AK is still active in military, still produce in mass.  both guns are extremely old, Yes S.T.A.R is an acronym it stands for  "Special Tactics and Rescue Service". Its a Resident Evil Reference